### DISCUSSION TITLE
Calculator.xaml.cs

### DISCUSSION THREAD

**Discussion by haribo841** [Projekt-Inzynierski-AK-AS-FD/Projekt-Inzynierski#17] [Ideas]
Filippo, czy wiesz że, masz 102 ostrzeżenia w pliku?
W projekcie masz swoją własną, nową kolumnę poświęconą tylko temu plikowi.
Powinno Ci być łatwiej to ogarnąć, bo teraz ten temat nie miesza się z innymi.
Spokojnie, pomogę w organizacji i wszystko wypiszę!
Powrzucam ci te błędy do tabeli i będziesz mógł się skupić na ich naprawieniu.
https://github.com/Projekt-Inzynierski-AK-AS-FD/Projekt-Inzynierski/projects/1#column-17119387

**Comment by Filippo1998r** [comment]
okej, to co miałoby pomóc to porobię, ale najpierw bugi i te znaczące invalid

**Reply by astolowska** [reply]
Sprawdziłam postępy z poprzedniego tygodnia (nie licząc wczorajszego commita) i został wypracowany review.

**Comment by haribo841** [comment]
![image](https://user-images.githubusercontent.com/48011447/144882152-b22ff01c-53ef-4e7b-abd1-7406ba7c9cdf.png)
Kropkę dostaje w wyniku dzielenia, zarówno ze znakiem ':' jak i ułamkiem (1/x).
Jakiś problem z logiką, bo powinno wyjść 3,96. Zobacz, czy daje przecinek czy co...

**Reply by astolowska** [reply]
Jeśli zostało to naprawione, warto tu to oznajmić.

**Comment by Filippo1998r** [comment]
Dokładnie, komputer w tym momencie rozróżniał nam przecinek od kropki, ale po update już ich nie rozróżnia i problem naprawiony

**Reply by haribo841** [reply]
Super, wszystko teraz działa. Proponuje trzymać się podtematu poprzez odpowiedzi, a jak coś nowego odnośnie tematu, to w nowym komentarzu. W innych dyskusjach na gicie nadal nie odpowiedziałeś.
